### DISCUSSION TITLE
历史最疯狂的几天

### DISCUSSION THREAD

**Discussion by noberumotto** [Planshit/ProjectEye#63] [General]
😂

![Snipaste_2021-10-29_23-30-05](https://user-images.githubusercontent.com/45778927/139462287-1f870d0c-24d2-4551-bbe4-233219cc7237.jpg)

**Comment by qitiandashengsunwukong** [comment]
我月均300小时，卷不卷

**Reply by noberumotto** [reply]
这环境好难，我这个月也300个小时了
